### DISCUSSION TITLE
Question on Peng's algorithm

### DISCUSSION THREAD

**Discussion by FanwangM** [theochem/procrustes#179] [Q&A] [answered]
Here are some questions that @banrovegrie has,

1. What does $\ast$ denote in the definition of $\hat{S}$?
$$\hat{S} = \Phi \ast (U^T_1 F V_1 \Sigma + \Sigma V^T_1 F^TU_1)$$

2. Doesn't $\Phi \in \mathbb{R}^{r \times r}$ rather than $\mathbb{R}^{n\times n}$?

3. What does $\hat{P_{11}^{+}}$ in the paper? Am I supposed to take transpose conjugate or something (which doesn't make sense)? Also, $\hat{P_{11}}$ by default has all of its eigenvalues $\geq 0$.

**Comment by banrovegrie** [comment]
Ok, so the first problem is resolved. It is the Hadamard product itself. So, that is fine.

**Reply by banrovegrie** [reply]
@FanwangM both problems are solved. It was very silly from my side. Confirmed while reading the proof that $^+$ indeed denotes pseudoinverse.

**Reply by FanwangM** [reply]
No worries at all. I am happy that you resolved them by yourself. 

So, we only have the third question that's left to be answered?

**Reply by banrovegrie** [reply]
I think that it is actually supposed to be $\mathbb{R}^{r \times r}$. I assumed it within the implementation and my tests seem to be passing.

**Reply by banrovegrie** [reply, marked as answer]
@FanwangM I think we can consider all the questions as resolved.
